.github/workflows/coverage-diff.yml

```diff
@@ -0,0 +1,35 @@
+name: Check coverage for PR
+
+on:
+  pull_request:
+
+jobs:
+  run-tests-check-coverage:
+    runs-on: ubuntu-20.04
+    name: Run tests & check coverage
+    steps:
+      - uses: actions/checkout@v3
+      - name: Jest coverage comment
+        id: coverage
+        uses: ArtiomTr/jest-coverage-report-action@9f733792c44d05327cb371766bf78a5261e43936
+        with:
+          package-manager: yarn
+          output: report-markdown
+      - name: Read coverage text report
+        id: coverage-text
+        shell: bash
+        run: |
+          OUTPUT=$(sed -e '1d' -e '$d' coverage/coverage.txt)
+          OUTPUT="${OUTPUT//'%'/'%25'}"
+          OUTPUT="${OUTPUT//$'\n'/'%0A'}"
+          OUTPUT="${OUTPUT//$'\r'/'%0D'}"
+          echo "::set-output name=content::$OUTPUT"
+      - uses: marocchino/sticky-pull-request-comment@adca94abcaf73c10466a71cc83ae561fd66d1a56
```

Comment:
It looks like "::set-output" is deprecated now: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/